### ISSUE TITLE
Index for all channels not refreshing on mambabuild

### ISSUE THREAD

**Issue by gabrielcnr** [mamba-org/boa#80]
The index repodata cache is not being refreshed for all the conda channels because of the big ttl set on:

https://github.com/mamba-org/boa/blob/fc9050d1f392382b09571e72cffc8554d3003802/boa/cli/mambabuild.py#L71

I think this need to be configurable or, at least, much, much smaller.

@wolfv @marcelotrevisani - what do you think?

I searched at `mamba`'s codebase and the ttl of the cache doesn't seem to be set to a high number there, so it's a problem only for `mambabuild`.

**Comment by marcelotrevisani** [comment]
I believe that would be nice to have it at least configurable in a certain way
maybe something like
```mamba --config local_repodata_ttl=1000```

**Comment by gabrielcnr** [comment]
Is it possible to use `conda` config for that?

For example:

`conda config --set boa.local_repodata_ttl 600`

Or conda would not allow it because:

```
CondaValueError: Key 'boa' is not a known primitive parameter.
```

Does this mean we would have something like a `.mambarc` or `.boarc` ?

Can this be passed as a command-line argument to `mambabuild` for now?

I guess we need your input here @wolfv  :-)

**Comment by wolfv** [comment]
Yeah I'd be happy to have a boarc file! Will need it soon for other stuff. PRs welcome :)

**Comment by gabrielcnr** [comment]
@wolfv - I'm happy to contribute with the `.boarc` :-)
Do you think we should leverage the Config machinery available from `conda` (since conda is already a dependency) or implement a new one?

**Comment by wolfv** [comment]
Good question. Might be better to start a new one, I'm not sure how conda handles unknown keys
